```
>           assert cluster.request.call_args == call(
                False, LOCK_DOOR, (), expect_reply=True, manufacturer=None)
E           AssertionError: assert call(False, 0...facturer=None) == call(False, 0,...facturer=None)
E           AssertionError: assert call(False, 0...facturer=None) == call(False, 0,...facturer=None)
E             At index 0 diff: (False, 0, (<class 'zigpy.types.basic.Optional.<locals>.Optional'>,)) != ''
E             Right contains one more item: {'expect_reply': True, 'manufacturer': None}
E             Full diff:
E             - call(False, 0, (<class 'zigpy.types.basic.Optional.<locals>.Optional'>,), expect_reply=True, manufacturer=None)
E             + call(False, 0, (), expect_reply=True, manufacturer=None)
```

third param of a `Cluster.request()` method is schema. I think the test broke once the zigpy DoorLock cluster schema was updated for that command, regardless of schema content the assertion would fail, since schema for the command is not empty anymore.

I'm personally (cause that how I learnt it from `bellows`) in favor  in checking just the required args, thus no need to update the test in the underlying API changes.
eg make it into two assertions, and check command_id and command_args (if any), lile:
```
assert cluster.request.call_args[0][0] is False
assert cluster.request.call_args[0][1] == LOCK_DOOR
```
and since no args were given to command, no *args to check. HA tests should not be concerned that `request()` method gets the right schema or any params internal to zigpy.
 
Also, specifically for this case IMO it is better to mock the `command()` method of the cluster class and check call_args for this method, to keep tests from breaking if zigpy api changes

--- a/tests/components/zha/test_lock.py
+++ b/tests/components/zha/test_lock.py
@@ -1,5 +1,5 @@
 """Test zha lock."""
-from unittest.mock import call, patch
+from unittest.mock import patch
 from homeassistant.const import (
     STATE_LOCKED, STATE_UNLOCKED, STATE_UNAVAILABLE)
 from homeassistant.components.lock import DOMAIN

--- a/tests/components/zha/test_lock.py
+++ b/tests/components/zha/test_lock.py
@@ -69,8 +69,8 @@ async def async_lock(hass, cluster, entity_id):
             'entity_id': entity_id
         }, blocking=True)
         assert cluster.request.call_count == 1
-        assert cluster.request.call_args == call(
-            False, LOCK_DOOR, (), expect_reply=True, manufacturer=None)
+        assert cluster.request.call_args[0][0] is False
+        assert cluster.request.call_args[0][1] == LOCK_DOOR
 
 
 async def async_unlock(hass, cluster, entity_id):
@@ -84,5 +84,5 @@ async def async_unlock(hass, cluster, entity_id):
             'entity_id': entity_id
         }, blocking=True)
         assert cluster.request.call_count == 1
-        assert cluster.request.call_args == call(
-            False, UNLOCK_DOOR, (), expect_reply=True, manufacturer=None)
+        assert cluster.request.call_args[0][0] is False
+        assert cluster.request.call_args[0][1] == UNLOCK_DOOR

--- a/tests/components/zha/test_lock.py
+++ b/tests/components/zha/test_lock.py
@@ -55,7 +55,7 @@ async def test_lock(hass, config_entry, zha_gateway):
     await async_lock(hass, cluster, entity_id)
 
     # unlock from HA
-    await async_lock(hass, cluster, entity_id)
+    await async_unlock(hass, cluster, entity_id)
 
 
 async def async_lock(hass, cluster, entity_id):

